frontend/app/routes/_public+/apply+/$id+/confirm.tsx

```diff
@@ -1,38 +1,148 @@
+  pageTitleI18nKey: 'apply:confirm.page-title',
+} as const satisfies RouteHandleData;
+
 export async function loader({ request, params }: LoaderFunctionArgs) {
-  const applyFlow = getApplyFlow();
-  const { id, state } = await applyFlow.loadState({ request, params });
-  return json({ id, state });
-}
+  //TODO: Get User/apply form information
 
-export async function action({ request, params }: ActionFunctionArgs) {
-  const applyFlow = getApplyFlow();
-  await applyFlow.loadState({ request, params });
-  const sessionResponseInit = await applyFlow.clearState({ request, params });
-  return redirectWithSuccess('/', 'Form Submitted!', sessionResponseInit);
+  const userInfo = {
+    firstName: 'John',
+    id: '00000000-0000-0000-0000-000000000000',
+    lastName: 'Maverick',
+    phoneNumber: '(754) 628-4776',
+    altPhoneNumber: '(819) 777-1234',
+    preferredLanguage: 'fr',
+    birthday: new Date().toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
+    sin: '123456789',
+    martialStatus: 'Married',
+    email: 'myemail@example.com',
+    communicationPreference: 'email',
+  };
+  const spouseInfo = {
+    firstName: 'Phil',
+    id: '00000000-0000-0000-0000-000000000001',
+    lastName: 'Doe',
+    phoneNumber: '(754) 628-4776',
+    birthday: new Date().toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
+    sin: '123456789',
+    consent: true,
+  };
+  const preferredLanguage = {
+    id: 'fr',
+    nameEn: 'French',
+    nameFr: 'Français',
+  };
+  const mailingAddressInfo = {
+    address: '44367 Ibrahim Field',
+    city: 'Lake Granvillestead',
+    province: 'PE',
+    postalCode: 'P1L 1C5',
+    country: 'CAN',
+  };
+  const homeAddressInfo = {
+    address: '724 Mason Mission',
+    city: 'Reichelmouth',
+    province: 'NS',
+    postalCode: 'T9K 6P4',
+    country: 'CAN',
+  };
+  const dentalInsurance = {
+    private: [],
+    public: [{ id: 'benefit-id', benefitEn: 'Dental and Optcial Assistance for Senioirs (65+)', benefitFR: '(FR) Dental and Optcial Assistance for Senioirs (65+)' }],
+  };
+
+  return json({ userInfo, spouseInfo, preferredLanguage, homeAddressInfo, mailingAddressInfo, dentalInsurance });
 }
 
 export default function ApplyFlowConfirm() {
-  const { id, state } = useLoaderData<typeof loader>();
+  const { userInfo, spouseInfo, preferredLanguage, homeAddressInfo, mailingAddressInfo, dentalInsurance } = useLoaderData<typeof loader>();
+  const { i18n, t } = useTranslation(i18nNamespaces);
+
+  const MSCALink = <InlineLink to={`https://www.canada.ca/${i18n.language === 'en' ? 'en/employment-social-development/services/my-account.html' : 'fr/emploi-developpement-social/services/mon-dossier.html'}`} />;
+  const dentalContactUsLink = <InlineLink to={`https://www.canada.ca/${i18n.language === 'en' ? 'en/services/benefits/dental/dental-care-plan/contact.html' : 'fr/services/prestations/dentaire/regime-soins-dentaires/contactez.html'}`} />;
```

Comment:
We should define the full URLs (including the `https://www.canada.ca/` part) in the i18n files. That way we won't have to check for the current language and we can reference the i18n key instead.